lib/stages/mergeBeamBuffer.hpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brife Merge the single frame beam buffer to a buffer with longer timespan.
+ *  - mergeBeamBuffer : public kotekan::Stage
+ */
+
+#ifndef MERGE_BEAM_BUFFER
+#define MERGE_BEAM_BUFFER
+
+#include "Config.hpp" // for Config
+#include "Stage.hpp"
+#include "bufferContainer.hpp" // for bufferContainer
+#include "BeamMetadata.hpp"   // for BeamMetadata
+
+#include <stdint.h> // for uint32_t
+#include <string>   // for string
+#include <vector>   // for vector
+
+
+using std::vector;
+
+
+/**
+ * @class mergeBeamBuffer
+ * @brief A stage to merge n single beam frame to one single 
+ *        beam frame.
+ *
+ * @par Buffers
+ * @buffer in_buf Kotekan buffer of raw packets.
+ *     @buffer_format Array of @c chars
```

Comment:
Include a line for each buffer the metadata type.
```suggestion
 *     @buffer_format Array of @c chars
 *     @buffer_metadata MergedBeamMetadata
```